integ-test/src/test/java/org/opensearch/sql/calcite/standalone/CalcitePPLBuiltinFunctionIT.java

```diff
@@ -20,16 +24,303 @@ public void init() throws IOException {
     loadIndex(Index.STATE_COUNTRY_WITH_NULL);
   }
 
+  private static boolean containsMessage(Throwable throwable, String message) {
```

Comment:
minor: use `verifyErrorMessageContains`

Reply:
One problem in this IT is all the test cases are applying on literals instead of fields.
Could you add some IT for fields with 
```
    loadIndex(Index.DATA_TYPE_NUMERIC);
    loadIndex(Index.DATA_TYPE_NONNUMERIC);
```